### DISCUSSION TITLE
How can I determine if I have the Student Developer Pack?

### DISCUSSION THREAD

**Discussion by rhit-parsonjc** [community/community#127767] [GitHub Education] [answered]
### Hi everyone,

I saw the PRO label on my profile despite being on the GitHub Free plan according to https://github.com/settings/billing/summary. I asked about this on https://github.com/orgs/community/discussions/127152, and Adnan25z suggested that I might have the Student Developer Pack. Is there a way to verify that this is the case?

**Comment by davevad93** [comment, marked as answer]
Hi @rhit-parsonjc ,

Yes, you can check your student benefits by going [here](https://education.github.com/discount_requests/application), if you see a status card that states `Congrats! Your request for @rhit-parsonjc was approved and your benefits are now available.` that means you have access to the Student Developer Pack.

**Reply by rhit-parsonjc** [reply]
I submitted a request on February 25, 2020, and it was approved. How long does the Student Developer Pack last before it needs to be renewed?

**Reply by davevad93** [reply]
The student validation lasts for 2 years, after which you will need to revalidate your student status. Yours ended on February 25, 2022, so you have to reapply to renew your benefits.

**Reply by rhit-parsonjc** [reply]
So the PRO label stays on the profile even after the student pack expires?

**Reply by davevad93** [reply]
As far as I know no, it should disappear from the profile once the benefits expire.

**Reply by redmuschket** [reply]
Возможно у вас по старым правилам льгота. Например мне JetBrean выдал студенческую лицензию. Я досихпор пользуюсь их продуктами, но после ввода санкций у меня не обновляется ПО. Посколько Git в этом плане нейтральние, у вас бессрочный доступ по старым правилам без ограничений

**Reply by ratish-rawat** [reply]
![Screenshot_2025-05-12-01-27-30-219_com android chrome](https://github.com/user-attachments/assets/720c1564-411f-405c-80b3-b6f13b7ab397)


So, when it says approved does it mean I got the student pack or is it that only the application request has been sent and I'll have to wait for the approval?

**Reply by HasanRifat22** [reply]
same here dont understand if i have the pack

**Reply by rizky-chy** [reply]
<img width="1381" height="611" alt="image" src="https://github.com/user-attachments/assets/cab427e9-be00-45ae-81c8-bdebc49f70cb" />
is this mean that i got approved already? my friend got it also but i got different beneficial from him. he got much more copilot model than me. and also the interface he got on the copilot menu is much different than mine:
<img width="1400" height="699" alt="image" src="https://github.com/user-attachments/assets/b7aaf51e-a0c6-4be8-896c-ac5b44989799" />

**Reply by mqordhowi0** [reply]
me too broo. my friend got on the copilot menu is much different than mine
